storage/mysql/tree_storage.go

```diff
@@ -711,6 +712,20 @@ func (t *tx) removeSequencedLeaves(leaves []trillian.LogLeaf) error {
 	return nil
 }
 
+// TODO: This only works for sizes where there is a stored tree head. This is deliberate atm
+// as serving proofs at intermediate tree sizes is complicated and will be implemented later.
+func (t *tx) GetTreeRevisionAtSize(treeSize int64) (int64, error) {
```

Comment:
Cut and pasted, looks like an opportunity for it to get out of date though. Inheriting docs was probably another NIH thing.